library/core/src/macros/panic.md

```diff
@@ -64,6 +64,25 @@ For more detailed information about error handling check out the [book] or the
 If the main thread panics it will terminate all your threads and end your
 program with code `101`.
 
+# Editions
```

Comment:
Thanks, having this written out here is really handy :+1:

One request: Can you put the information about the *newest* edition first in this section?  We want to optimize the docs for people using the latest edition, and have people still using older ones need to scan down to find their behaviour, rather than having people using 2021 need to hunt.

Maybe it could work well to make subsections here?  Sketch,

```
# Editions

## 2021 and later

... `panic!` always uses formatting; use `panic_any` if you want a value ...

## 2018 and 2015

... panics with a payload of literally ...
```